spark/src/test/scala/org/apache/comet/CometCastSuite.scala

```diff
@@ -665,6 +665,30 @@ class CometCastSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     castTest(generateTimestamps(), DataTypes.DateType)
   }
 
+  test("cast short to byte") {
```

Comment:
These methods already exist in main but have different naming, so I think you need to upmerge/rebase against main.

Example:

```
ignore("cast ShortType to ByteType")
```